src/shared/prerendered-app/util.css

```diff
@@ -16,9 +16,5 @@
     font: inherit;
     padding: 0;
     margin: 0;
```

Comment:
```suggestion
    margin: 0;
    outline-offset: 10px;
```

Reply:
<img width="722" alt="Screenshot 2020-12-16 at 17 14 21" src="https://user-images.githubusercontent.com/1889710/102374906-27457200-3fc2-11eb-8382-7f109123c925.png">
Decided to go this route for focus styles. :)

Reply:
@argyleink Do you want me to add that `outline-offset: 10px` line to this PR?

Reply:
not needed anymore with your enhanced box-shadow solution, will remove 👍🏻